docs/deployments/deploy-to-vercel.mdx

```diff
@@ -21,57 +21,57 @@ To set up a new Vercel project, start by [going to your Vercel Dashboard](https:
 
-Once done, you'll see the option to import a Git repository. Find the repository you want to use and press "Import":
+Once done, you'll see the option to import a Git repository. Find the repository you want to use and press **Import**:
 
-![An "Import Git Repository" screen provides a few options, including selecting an organization and list of related repositories to import](/docs/images/deployment-guides/vercel-import-git-repository.png)
+![An **Import Git Repository** screen provides a few options, including selecting an organization and list of related repositories to import](/docs/images/deployment-guides/vercel-import-git-repository.png)
 
-Once you've selected a project to deploy, you'll be directed to the "Configure Project" page.
+Once you've selected a project to deploy, you'll be directed to the **Configure Project** page.
 
-On this page, **make sure to add "[Environment Variables](https://vercel.com/docs/concepts/projects/environment-variables)" from [your app's Clerk dashboard](https://dashboard.clerk.com/)** so that your Vercel project will be connected to your Clerk app properly:
+On this page, make sure to add **[Environment Variables](https://vercel.com/docs/concepts/projects/environment-variables)** from [your app's Clerk dashboard](https://dashboard.clerk.com/) so that your Vercel project will be connected to your Clerk app properly:
 
-![The "Configure Project" page allows you to add environment variables, select a framework preset, and more](/docs/images/deployment-guides/vercel-configure-project.png)
+![The **Configure Project** page allows you to add environment variables, select a framework preset, and more](/docs/images/deployment-guides/vercel-configure-project.png)
 
 The API keys added on this screen will be assigned to all deployments on Vercel, including preview URLs. To connect your first deployment to your Clerk app, be sure to **use your Clerk production API keys rather than your development API keys**.
 
 > [You'll need to take additional steps to make sure Vercel preview URLs work as intended.](#support-vercel-preview-urls)
 
-Once you press "Deploy", your app will deploy to production!
+Once you press **Deploy**, your app will deploy to production!
 
 > You will likely want to [assign a custom domain to your Vercel project](https://vercel.com/docs/concepts/projects/domains/add-a-domain) once you've completed this step.
 
-## Add Clerk API Keys to Existing Vercel Project
+## Add Clerk API keys to existing Vercel project
 
 To add your Clerk API keys to an existing Vercel project, start by selecting your project [from your Vercel dashboard](https://vercel.com/dashboard):
 
-![A card for the "clerk-nextjs-starter" project on our Vercel dashboard](/docs/images/deployment-guides/vercel-existing-project.png)
+![A card for the **clerk-nextjs-starter** project on our Vercel dashboard](/docs/images/deployment-guides/vercel-existing-project.png)
```

Comment:
This is in image alt text so I don't think the markdown styling applies here:
```suggestion
![A card for the "clerk-nextjs-starter" project on our Vercel dashboard](/docs/images/deployment-guides/vercel-existing-project.png)
```

Reply:
Good catch - I think I was scrolling for " " and accidentally changed that one!